soc/nxp/imxrt/imxrt6xx/adsp/include/adsp/io.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INCLUDE_IO__
```

Comment:
Is this stuff still used?

Reply:
Looks like this is a relict from SOF world. For now it is not used. Will add this back once there will be users for it! Will remove it in v3.